airqo-platform/AirQo-frontend
# [NetManager] Integrate auth service tenancy

#### Summary of Changes (What does this PR do?)
- Enables auth-service tenancy support

#### Is this change ready to hit production in its current state?

#### Status of maturity (all need to be checked before merging):


Good to have (up to reviewer's discretion):

#### How should this be manually tested?
* Pull and checkout to this [branch]() locally
* Install node requirements `npm install`
* Ensure the local auth-service with data-segregation is running. Check this [PR](https://github.com/airqo-platform/AirQo-api/pull/121) for more info
* You start the application in one of the two ways:
  1. `npm run dev-mac` (on mac) or `npm run dev-pc` (on windows platform)
  2. `npm run prod-mac` (on mac) or `npm run prod-pc` (on windows platform) and manually change the auth-service url configs to point to the locally running service. The benefit of this approach is you are able to load other data required by the app (such as analytics).
* On the login page, a new field `organization` is required. Using the current credentials and organization, login should be successful.

#### What are the relevant tickets?
- [PLAT-164](https://airqoteam.atlassian.net/browse/PLAT-164)

#### Screenshots (optional)
* The login page screenshot
<img width="1170" alt="Screenshot 2020-09-23 at 15 48 51" src="https://user-images.githubusercontent.com/39955305/94015519-a8bba880-fdb5-11ea-8e76-28779d9bf0d6.png">
